From 38f23d7f3a0a7e5854efb84f768f5237fa0962c5 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Tue, 12 Mar 2019 17:40:29 -0600
Subject: [PATCH 2/2] schemas/dash-to-dock: set dash-max-icon-size to 42

Because 42 it's the ultimate answer, we want icon size
to be this. Jokes aside, this is change requested from
internal design feedback.

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 .../org.gnome.shell.extensions.dash-to-dock.gschema.override    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override b/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override
index 287d035..a82e55e 100644
--- a/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override
+++ b/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override
@@ -4,7 +4,7 @@ click-action='minimize-or-overview'
 custom-background-color=false
 custom-theme-customize-running-dots=false
 custom-theme-shrink=false
-dash-max-icon-size=64
+dash-max-icon-size=42
 dock-position='LEFT'
 force-straight-corner=false
 max-alpha=0.8
-- 
2.20.1

